page/about.md

```diff
@@ -12,7 +12,7 @@ The process for designing and planning the future of the jQuery UI library shoul
 
 ### Flexible styling and themes
 
-Widgets should provide hooks to enable developers to customize both behavioral and presentational aspects. Transition animations should be optional and customizable. Class names used on internal elements should be meaningful to jQuery UI users and enable styling either through [ThemeRoller](http://jqueryui.com/themeroller/) or hand-written CSS via the [jQuery UI CSS Framework](http://learn.jquery.com/jquery-ui/theming/). As much as possible, style attributes should be separated into the plugin's CSS, not within the scripts in order to make customization simple and clear. Widgets styles should be coded with proportional (em-based or % based) sizing and should re-flow horizontally to fill the space provided.
+Widgets should provide hooks to enable developers to customize both behavioral and presentational aspects. Transition animations should be optional and customizable. Class names used on internal elements should be meaningful to jQuery UI users and enable styling either through [ThemeRoller](https://jqueryui.com/themeroller/) or hand-written CSS via the [jQuery UI CSS Framework](https://learn.jquery.com/jquery-ui/theming/). As much as possible, style attributes should be separated into the plugin's CSS, not within the scripts in order to make customization simple and clear. Widgets styles should be coded with proportional (em-based or % based) sizing and should re-flow horizontally to fill the space provided.
```

Comment:
We need to be careful with changes to anything related to download builder because the resources are managed in a different repo. We'll need download.jqueryui.com to be updated first, or at least at the same time.